elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -1,7 +1,113 @@
 defmodule Web.Settings.DNS do
   use Web, :live_view
 
+  alias Domain.Config
```

Comment:
Please remove a newline before this one, so that all use/import/require blocks are together.

Reply:
👍